src/main/groovy/3shutdownCLI.groovy

```diff
@@ -60,12 +60,15 @@ if (cliEnabled) {
     System.exit(0)
 }
 
+// disable remoting cli
+jenkins.getDescriptor("jenkins.CLI").get().setEnabled(false)
```

Comment:
Not a blocker, but shouldn't this be enough? I'm just curious what this option does.

Reply:
So you can connect to the cli with either -remoting -http or -ssh. This option disables the -remoting option but if you go to the cli via manage jenkins > cli that endpoint is still available. More here: https://jenkins.io/doc/book/managing/cli/